src/io.cpp

```diff
@@ -692,7 +692,7 @@ bool tilde(const char *file, char *expanded) {
 
     if (*file == '~') {
         char user[128];
-        struct passwd *pw = NULL;
+		struct passwd *pw = nullptr;
```

Comment:
Although this `nullptr` is a good change, I would rather not introduce new changes for the PR. We can keep this change, but perhaps save any other changes for your next PR :)

Also, I've been using spaces for indentation, and would prefer to keep consistency, so can you please change this. I've noticed this in several other places so can you please double check your previous commits for those others -- I'll also go and update the [CONTRIBUTING](https://github.com/dungeons-of-moria/umoria/blob/master/CONTRIBUTING.md) guidelines to note this requirement :)